source/location_access/locacc_lost_woods.cpp

```diff
@@ -312,6 +364,14 @@ void AreaTable_Init_LostWoods() {
                  // Locations
                  LocationAccess(LW_DEKU_SCRUB_NEAR_DEKU_THEATER_RIGHT, { [] { return IsChild && CanStunDeku; } }),
                  LocationAccess(LW_DEKU_SCRUB_NEAR_DEKU_THEATER_LEFT, { [] { return IsChild && CanStunDeku; } }),
+                 LocationAccess(LW_IN_BOULDER_BLUE_RUPEE,
+                                { [] { return Here(LW_BEYOND_MIDO, [] { return CanBlastOrSmash; }); },
```

Comment:
Here() check is unnecessary as the age that destroys the boulder can also collect the item. Logically this is fine but it'll add some time to generation. (Note the same isn't true for grotto boulders as the entrance remaining open is important)
```suggestion
                                { [] { return CanBlastOrSmash; },
```